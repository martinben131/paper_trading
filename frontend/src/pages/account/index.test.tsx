import React from "react";
import { shallow } from "enzyme";
import Profile from "./index";

test("Renders profile correctly", () => {
  const component = shallow(<Profile />);
  expect(component).toMatchSnapshot();
});
