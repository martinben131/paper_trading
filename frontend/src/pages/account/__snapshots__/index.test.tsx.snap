// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders profile correctly 1`] = `
<Fragment>
  <Balance />
  <br />
  <History />
  <br />
  <Profile />
</Fragment>
`;
